docs/integrations-in-rancher/rancher-extensions.md

```diff
@@ -41,17 +41,41 @@ In v2.7.0, the built-in extensions will not be displayed under the **Available**
 
     ![Manage repositories](/img/manage-repos.png)
 
-5. Under the **Available** tab, click **Install** on the desired extension and version as in the example below. Note that you can easily update your extension as the button to **Update** will appear on the extension if one is available.
+5. Under the **Available** tab, click **Install** on the desired extension and version as in the example below. You can also update your extension from this screen, as the button to **Update** will appear on the extension if one is available.
 
     ![Install Kubewarden](/img/install-kubewarden.png)
 
 6. Click the **Reload** page button that will appear after your extension successfully installs. Note that a logged-in user who has just installed an extension will not see a change to the UI **unless** they reload the page.
 
     ![Reload button](/img/reload-button.png)
 
+### Importing and Installing Extensions in an Air-Gapped Environment
```

Comment:
Nit: Importing and Installing Extensions Repository in an Air-Gapped Environment